templates/base.html

```diff
@@ -83,14 +83,18 @@ <h1 class="display-4 text-center">Where are you?</h1>
           <div class="col-12 col-md text-center">
             <small class="d-block mb-3">
               <a href="{% url "tos" %}">Terms of Service</a>
+              •
               <a href="{% url "about" %}">About</a>
             </small>
           </div>
         </div>
         <div class="row">
           <div class="col-12 col-md text-center">
-            <small class="d-block mb-3">
-              <a href="https://ko-fi.com/B0B586NC">Support us on Ko-fi</a>
+            <small class="d-block">
+              This website is <a href="https://github.com/wlonk/wheretofind.me"><span class="fab fa-github"></span> open-source</a>. <br>
+              Help us make it better by <a href="https://github.com/wlonk/wheretofind.me/issues">suggesting changes</a>. Or even better - implement them. </br>
```

Comment:
How about

```html
<a href="https://github.com/wlonk/wheretofind.me/issues">Help us make it better!</a>
```

Breaking it down between issues and pull requests in this call to action feels weirdly over-specific to me.

Reply:
you're probably right. but that whole section feels kind of weird to me. first, there is canonical link to the repository, then there's this call-to-action, then there's a feedback link that point's to wlonk's profile. I thought I'd put them all in anyway and make changes from feedback.

But this changeset was to brag about the website being open source. So in that spirit, I think it is okay this way. I'm not saying that it should be merged as is. There can be something that can be modified so that it doesn't feel like a desperate call-to-action or an over-done footer.